ex2/ex2.c

```diff
@@ -8,7 +8,26 @@
 
 int main(int argc, char* argv[])
 {
-    // Your code here 
+    FILE *fp;
+    char buffer[255];
+
+    fp = fopen("text.txt", "r");
+
+    pid_t rc = fork();
+
+    if (rc == 0)
+    {
+        fgets(buffer, 255, fp); //read one line
+        printf("Child read: %s", buffer);
```

Comment:
Program seems to read correctly, but you'll also want to use `fprintf` to write something back to the file.